### ISSUE TITLE
[BUR-67] Implement UI for client-side authentication with the Hack Club server

### ISSUE THREAD

**Issue by conradev** [hackclub/burrow#178]
<sub>From [SyncLinear.com](https://synclinear.com) | [BUR-67](https://linear.app/hackclub/issue/BUR-67/implement-ui-for-client-side-authentication-with-the-hack-club-server)</sub>

**Comment by conradev** [comment]
@conradev, we are closing this in favor of [BUR-105](https://linear.app/hackclub/issue/BUR-105/architect-authentication-scheme-using-slack) yea?

**Comment by conradev** [comment]
Unblocked.

**Comment by conradev** [comment]
Is this still blocked by #72?
